### DISCUSSION TITLE
How to open files on iCloud Drive

### DISCUSSION THREAD

**Discussion by splottdodger** [ttscoff/bunch-docs#137] [Q&A] [answered]
This looks to be a really excellent app, but before I donate I'd like to know if I can open iCloud Drive files in a bunch. I can't get it to work at the moment and get an error saying Bunch is unable to open the file, which is on my shared Desktop. Any ideas on how to solve this?

Chris

**Comment by ttscoff** [comment]
I would need more info to troubleshoot this. Is the issue with ANY file 
on your Desktop, or a particular file? And what app are you trying to 
open the file in?

-Brett

**Reply by splottdodger** [reply]
I am trying to open a file in TaskPaper called "Current Projects" in my Desktop Folder (shared via iCloud). I have tried different versions of the file path:

~/Desktop/Current Projects
~/Library/Mobile Documents/Desktop/Current Projects

I also tried quoting the above in case the spaces were causing problems and then renamed the file to CurrentProjects to avoid the space altogether. 

I get the following error message. If I don't try to open this file TaskPaper works fine.

<img width="266" alt="Screenshot 2021-08-17 at 11 59 36" src="https://user-images.githubusercontent.com/11377777/129714868-48d943da-e219-4ca2-a42c-4e7522b0a572.png">

**Reply by splottdodger** [reply]
My apologies. I have resolved this now: I had not included the filename extension (.taskpaper).

**Reply by ttscoff** [reply]
Ah, yeah, that'll do it :)

**Comment by splottdodger** [comment, marked as answer]
User error!
